src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -645,7 +651,7 @@ namespace winrt::TerminalApp::implementation
     //      currently displayed, it will be shown.
     // Arguments:
     // - settings: the TerminalSettings object to use to create the TerminalControl with.
```

Comment:
nit: update the comments up here for this function to say that you're returning a tab now